website/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@graphprotocol/docs",
+  "version": "1.0.0",
+  "packageManager": "pnpm@7.28.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "rm -rf .next && next build",
+    "export": "rm -rf out && next export -o ../out/docs",
```

```suggestion
    "export": "rm -rf ../out && next export -o ../out/docs",
```
Though, I wonder...
1. Should `out` be under `website`? (That would require changing some stuff in `Dockerfile`, but...)
2. Should `Dockerfile` be under `website`?

---

I think also `Dockerfile` should be in `website`!

---

it is simpler to keep on root. This makes it so that we can have docker context bootstrap from the root of mono-repo. We can build the full monorepo from root and then only have it copy files that matter in final image that we publish.